reload_app/events.py

```diff
@@ -1509,6 +1509,15 @@
         "conditions": list,
         "project_id": str,
     },
+    'sampling.settings.modal.specify.client.rate_cancel': {
+        "project_id": str,
+    },
+    'sampling.settings.modal.specify.client.rate_next': {
+        "project_id": str,
+    },
+    'sampling.settings.modal.specify.client.rate_read_docs': {
+        "project_id": str,
+    }, 
```

Comment:
```suggestion
    },
```

python -m flake8
./reload_app/events.py:1520:7: W291 trailing whitespace

Reply:
for some reason, I am unable to format this python file  locally :(

Reply:
thanks